test/unit/legacy_wrappers/collection.test.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const {
+  MongoClient,
+  FindCursor,
+  ListIndexesCursor,
+  ChangeStream,
+  AggregationCursor,
+  Collection
+} = require('../../../src/index');
+const mongodbDriver = require('mongodb');
+const sinon = require('sinon');
+const { expect } = require('chai');
+
+const iLoveJs = 'mongodb://iLoveJavascript';
+
+describe('legacy_wrappers/collection.js', () => {
+  let client;
+  let collection;
+  beforeEach(async function () {
+    client = new MongoClient(iLoveJs);
+    collection = client.db().collection('test');
+  });
+
+  afterEach(async function () {
+    sinon.restore();
+    await client.close();
+  });
+
+  it('should return legacy listIndexes cursor', () => {
+    expect(collection.listIndexes()).to.be.instanceOf(ListIndexesCursor);
+  });
+
+  it('should return legacy ChangeStream', () => {
+    expect(collection.watch()).to.be.instanceOf(ChangeStream);
+  });
+
+  it('should return legacy ChangeStream', () => {
+    expect(collection.aggregate()).to.be.instanceOf(AggregationCursor);
+  });
+
+  it('should return legacy FindCursor', () => {
+    expect(collection.find()).to.be.instanceOf(FindCursor);
+  });
+
+  it('should support rename(newName)', async () => {
```

Comment:
can we similarly improve these `it` statements to be more descriptive of what's actually being tested? clearly indicate success/failure expectations and all that jazz? (see my long comment on api.test.js)